src/plugins/dashboard/public/dashboard_app/_dashboard_app.scss

```diff
@@ -1,3 +1,13 @@
+@import 'src/core/public/mixins';
+
+.dshAppWrapper {
+  @include euiBreakpoint('m', 'l', 'xl') {
+    @include kibanaFullBodyHeight();
+  }
```

Comment:
Excluding the `kibanaFullBodyHeight` mixin from the `s` breakpoint appears to be causing dueling scrollbars to appear at small viewport sizes.

![image](https://user-images.githubusercontent.com/3884767/217590015-3af2647a-3b93-4977-a4a4-54036d7bec54.png)

Reply:
Oh nice catch! I added a `xs` too because I was still seeing the dual scrollbar when I shrank the viewport width even more.

<img width="389" alt="Screenshot 2023-02-08 at 12 13 16 PM" src="https://user-images.githubusercontent.com/1697105/217641049-294a7995-acc6-4b1f-b872-d46630974e62.png">